_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,288 @@
+|------|-------------------------------------|---------------------------------------------------------------------------|-------------------------------------------------------------------|---------------------------------------------------------------------------------------|---------------------|
+| 1.   | [GPL v2+][GPL v2]                   | [GPL v2+][GPL v2]                                                         | [GPL v2+][GPL v2]                                                 | This is the initial state of the MDAnalysis library license.                          | < v2.6.0            |
+| 2.   | [GPL v3+][GPL v3]                   | [GPL v2+][GPL v2]                                                         | [LGPL v2.1+][LGPL v2.1]                                           | As of commit [44733fc][] all new contributions are made under [LGPL v2.1+][LGPL v2.1] | v2.6.0+             |
+| 3.   | [LGPL v3+][LGPL v3]                 | [LGPL v2.1+][LGPL v2.1]                                                   | [LGPL v2.1+][LGPL v2.1]                                           | Once historical contributors have agreed to relicense their contributions.            | N/A                 |
+| 4.   | [LGPL v2.1+][LGPL v2.1]             | [LGPL v2.1+][LGPL v2.1]                                                   | [LGPL v2.1+][LGPL v2.1]                                           | Once Apache 2.0 dependencies have been removed from the library.                      | N/A                 |
+
+In this section, we will briefly go over the various steps we propose to take (Table 2) as we go about relicensing the
+[MDAnalysis core library][].
+
+### 1. A GPL v2+ package and source code (MDAnalysis v2.5 and lower)
+
+This is the original state of the license up until the [2.5.x releases of MDAnalysis](https://www.mdanalysis.org/blog/page2/#releases-2-5-0-and-2-4-x-of-mdanalysis).
+The package and source code were released under a [GPL v2+][GPL v2] license.
+
+### 2. A GPL v3+ package, and a mix of historical GPL v2+ and new LGPL v2.1+ source code (MDAnalysis v2.6 onwards)
+
+As of commit [44733fc][], all new contributions to MDAnalysis have been made under the terms of the
+[LGPL v2.1+][LGPL v2.1] and the package is released under [GPL v3+][GPL v3] to better reflect our
+understanding of the [Apache 2.0 incompatibility issues](#why-the-interim-gpl-v3-and-lgpl-v3-package-licenses-the-issue-with-apache-20).
+Historical source code contributions remain under the terms of [GPL v2+][GPL v2].
+
+This will remain the state of MDAnalysis until we obtain approval from our developers to relicense
+their code to [LGPL v2.1+][LGPL v2.1]. As we obtain contributor agreements, these historical
```

Comment:
How is this obtained?

Reply:
I'll link to the section below

Reply:
This comment and the above are on the right place. The comments below are shifted (possibly due to an GitHub iOS App bugs...). Sorry for the confusion!